src/i3pager.cpp

```diff
@@ -51,6 +51,7 @@ QList<Workspace> I3Pager::getWorkspaces(bool filterByCurrentScreen, QString orde
         i3ipc::connection conn;
         auto i3workspaceList = conn.get_workspaces();
         qDebug() << "Loading workspaces:";
+        qDebug() << "Variables,CurrentScreen:"<<filterByCurrentScreen<<", orderBy:"<<orderWorkspacesBy;
```

Comment:
I suppose this was meant to be this?

```suggestion
        qDebug() << "Variables: CurrentScreen:" << filterByCurrentScreen << ", orderBy:" << orderWorkspacesBy;
```

Reply:
Yes, but you can even remove this line completely if you want. I just used it to understand the configuration parameters, after all it is my first plasmoid.